Sources/Testing/Traits/Comment.swift

```diff
@@ -150,3 +150,17 @@ extension Test {
       .flatMap(\.comments)
   }
 }
+
+extension Test.Snapshot {
+
+  /// The complete set of comments about this test from all traits of a certain
+  /// type.
+  ///
+  /// - Parameters:
+  ///   - traitType: The type of ``Trait`` whose comments should be returned.
+  ///
+  /// - Returns: The comments found for the specified test trait type.
+  public func comments<T>(from traitType: T.Type) -> [Comment] where T: Trait {
```

Comment:
This function exists so that we can filter comments based on the traits that add them, but since trait types are lost in the snapshot, it can't function correctly. We shouldn't include it.